tools/orchestrator/src/download.ts

```diff
@@ -91,7 +92,8 @@ function buildSonarQubeUrl(version: string) {
   return `https://repox.jfrog.io/repox/sonarsource/org/sonarsource/sonarqube/sonar-application/${version}/sonar-application-${version}.zip`;
```

Comment:
imo, it would be nice to move `https://repox.jfrog.io` part to the constants , actually it should be probably requested from env variable `ARTIFACTORY_URL` which is usually configured in CI, and only default to the hardcoded URL. Also it should be used for `VERSIONS_URL` const

Reply:
done